.github/workflows/Tests.yaml

```diff
@@ -216,7 +216,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        compiler: [gcc-7, gcc-8, gcc-9, clang-5.0, clang-8, clang-9]
+        compiler: [gcc-7,gcc-8,gcc-9,clang-6.0,clang-8,clang-9,clang-10]
```

Comment:
If you removed the spaces to keep the line length within 80 chars: You can just add a line break within the `[]` list.